packages/reputation-miner/ReputationMiner.js

```diff
@@ -375,7 +381,7 @@ class ReputationMiner {
     const colonyAddress = key.slice(2, 42);
     const skillId = key.slice(42, 106);
     const userAddress = key.slice(106);
-    return [colonyAddress, skillId, userAddress];
+    return [`0x${colonyAddress}`, `0x${skillId}`, `0x${userAddress}`];
```

Comment:
How did this pass build if `saveCurrentState` is still converting these https://github.com/JoinColony/colonyNetwork/blob/develop/packages/reputation-miner/ReputationMiner.js#L783

Reply:
You've done a rebase here, right? I think it was written before `saveCurrentState` was merged in.

Reply:
Ah I've broken it! Do you want me to fix?